lib/Collection.php

```diff
@@ -35,6 +35,11 @@ public function retrieve($id, $params = null, $opts = null)
         return Util\Util::convertToStripeObject($response, $opts);
     }
 
+    public function autoPagingIterator()
```

Comment:
Can we just add some basic function document on this one? We don't have any of the auto-paging stuff in our API docs right now, so it kind of makes sense to at least have some short documentation somewhere.